src/main/kotlin/no/nav/syfo/huskelapp/domain/Huskelapp.kt

```diff
@@ -0,0 +1,17 @@
+package no.nav.syfo.huskelapp.domain
+
+import no.nav.syfo.domain.PersonIdent
+import no.nav.syfo.personstatus.domain.PersonOversiktStatus
+import java.util.UUID
+
+data class Huskelapp(
+    val uuid: UUID,
+    val personIdent: PersonIdent,
+    val isActive: Boolean,
+) {
+    fun toPersonoversiktStatus() = PersonOversiktStatus(
+        fnr = personIdent.value,
+    ).copy(
+        huskelappActive = isActive,
+    )
```

Comment:
Gjorde sånn i stedet for å skrive det store objektet, nå som vi har default-values.
Hadde det vært penere med en `.create()`-metode? I så fall burde man kanskje lage flere, for aktivitetskrav, huskelapp, dialogmøtestatusendring etc etc.

Reply:
:+1: Hadde nok vært fint og ryddet litt opp i det ja, men får ta det i en egen PR.